packages/article-feed-persoo/README.md

```diff
@@ -0,0 +1,46 @@
+# Shopsys Category Feed Persoo
+
+[![Downloads](https://img.shields.io/packagist/dt/shopsys/article-feed-persoo.svg)](https://packagist.org/packages/shopsys/article-feed-persoo)
+
+Plugin for [Shopsys Platform](https://www.shopsys.com/shopsys-platform/) responsible for generating XML article feed for [Persoo](https://persoo.cz).
+
+For detailed information about the feed see [official documentation](https://support.persoo.ai).
+
+This repository is maintained by [shopsys/shopsys] monorepo, information about changes is in [monorepo CHANGELOG.md](https://github.com/shopsys/shopsys/blob/master/CHANGELOG.md).
+
+## Installation
+
+The plugin is a Symfony bundle and is installed in the same way:
+
+### Download
+
+Download the package using [Composer](https://getcomposer.org/):
+
+```
+composer require shopsys/category-feed-persoo
```

Comment:
this should be shopsys/article-feed-persoo, right?

Reply:
yeah, thank you, I see I was very inattentive, fixed in 69e50e3f43